base/docs/helpdb/Base.jl

```diff
@@ -1318,7 +1318,14 @@ bswap
 
 The largest integer losslessly representable by the given floating-point DataType `T`.
 """
-maxintfloat
+maxintfloat(T)
+
+"""
+    maxintfloat(T, S)
+
+The largest integer losslessly representable by the given floating-point DataType `T` that also does not exceed the maximum integer representable by the integer data type T.
```

Comment:
we prefer to wrap long lines in inline docstrings

the integer data type is the second input right? so one should say `S` with code highlighting